specs/mediawiki/v1/content.yaml

```diff
@@ -202,6 +221,19 @@ paths:
         # work in current Chrome and Firefox.
         # See also http://tools.ietf.org/html/rfc7231#section-7.1.2
         uri: /{domain}/sys/parsoid/html/{title}
+      x-amples:
+        - title: Should get a latest html of a page by title
+          request:
+            params:
+              title: Foobar
+            headers:
+              cache-control: no-cache
+          response:
+            status: 200
+            headers:
+              etag: /.+/
+              content-type: text/html
```

Comment:
@d00rman Could be both ways, in the test we check both a regexp and if the actual header starts with what's declared. Could expand this to full content-type with regexed version